tutorials/01_installation.md

```diff
@@ -93,7 +93,7 @@ sudo apt-get install git cmake pkg-config python ruby-ronn libignition-cmake2-de
 Clone the repository into a directory and go into it:
 
 ```
-git clone https://github.com/gazbosim/gz-fuel-tools /tmp/gz-fuel-tools
+git clone https://github.com/gazbosim/gz-fuel-tools /tmp/gz-fuel-tools -b ign-fuel-tools7
```

Comment:
```suggestion
git clone https://github.com/gazbosim/gz-fuel-tools /tmp/gz-fuel-tools -b -b ign-fuel-tools<#>
```

Reply:
Assuming there will be no other ign-fuel-tools branch, I'd vouch for the concrete version. Fortress is THE version that will be used by many for conversion from Classic, so let's make it as easy as possible :)

Reply:
I'm okay with putting the version number here, but users converting from Classic and building Gazebo from source should really using https://gazebosim.org/docs/fortress/install_ubuntu_src to build the whole collection from source. The instructions here are geared more toward users who want to use the library by itself.